packages/version.ts

```diff
@@ -1,2 +1,2 @@
 // this file gets auto-generated
-export const version = '5.24.2'
+export const version = '5.24.4'
```

Comment:
Yeah, this is a weird one.. The committed version is always one number behind, but it publishes correctly, so I didn't took the time to investigate a possible fix. As it gets auto generated, this file should not be changed. You can leave it, because CI will overwrite it anyways.

Reply:
Reverted